### ISSUE TITLE
[suggestion] use a subscription system like ublock does.

### ISSUE THREAD

**Issue by elypter** [diegocr/CleanLinks#141]
maybe you could even just copy that part of the code. that would make this addon a lot more flexible

**Comment by jawz101** [comment]
Agreed.  Way too many fixes end up in the comments here that should end up in the program itself
